pylint/checkers/imports.py

```diff
@@ -431,6 +431,15 @@ def visit_if(self, node):
         if is_nested_allowed and \
                 any(node.nodes_of_class((astroid.Import, astroid.ImportFrom))):
             return
+        if isinstance(node, astroid.Assign):
```

Comment:
FYI this code by @lmedioni have a similar case of your feedback with [def visit_functiondef](https://github.com/PyCQA/pylint/blob/4a3bd3e53df5ebec07b2edbf072e08b444b74b46/pylint/checkers/imports.py#L439-L459)

NOTE: I don't understand why we need 2 methods to compute the same first-non-import... with the same name of `visit_`
I can change that method too... but IMHO is offtopic for the initial focus of this PR (Maybe other one)